### PR TITLE
Updated namespaces of classes that didn't have one

### PR DESCRIPTION
**Pull request by StephenHodgson**
Classes that weren't in the HoloToolkit namespace.

Fixes: #1374

### CODE REVIEW

File: Assets/HoloToolkit/Utilities/Scripts/DisplayDependentObjectActivator.cs [hunk 1]
```diff
@@ -7,22 +7,25 @@
 using UnityEngine.XR.WSA;
 #endif
 
-public class DisplayDependentObjectActivator : MonoBehaviour
+namespace HoloToolkit.Unity
```
Comment (NeerajW) [inline Assets/HoloToolkit/Utilities/Scripts/DisplayDependentObjectActivator.cs]: Are any of the others under .Utilities?

File: Assets/HoloToolkit-Examples/Utilities/Scripts/CheckApiTest.cs [hunk 2]
```diff
@@ -4,26 +4,29 @@
 using UnityEngine;
 using UnityEngine.UI;
 
-public class CheckApiTest : MonoBehaviour
+namespace HoloToolkit.Unity.Tests
```
[the remark this answers is not in the document]
Comment (StephenHodgson) [reply Assets/HoloToolkit-Examples/Utilities/Scripts/CheckApiTest.cs]: I'm not sure what the difference is here.

File: Assets/HoloToolkit/UX/Scripts/Distorters/FastSimplexNoise.cs [hunk 3]
```diff
@@ -1,448 +1,589 @@
...
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
-//
 using System;
 
-/// <summary>
-/// A conglimeration of open-source simplex libraries in C# with an emphasis on performance
-/// </summary>
-public class FastSimplexNoise
+namespace HoloToolkit
```
[the remark this answers is not in the document]
Comment (StephenHodgson) [reply Assets/HoloToolkit/UX/Scripts/Distorters/FastSimplexNoise.cs]: That was a breaking change I was putting off.  The namespace needs to change in the main MixedRealityToolkit before we can change it here.

File: Assets/HoloToolkit/Utilities/Scripts/DisplayDependentObjectActivator.cs (hunk 1 above)
Comment (StephenHodgson) [reply Assets/HoloToolkit/Utilities/Scripts/DisplayDependentObjectActivator.cs]: No, they're all under `HoloToolkit.Unity`

File: Assets/HoloToolkit-Examples/MotionControllers-GrabMechanics/Scripts/ExtensionMethod.cs [hunk 4]
```diff
@@ -3,29 +3,31 @@
 
 #if UNITY_WSA && UNITY_2017_2_OR_NEWER
 using UnityEngine;
-using HoloToolkit.Unity.InputModule.Examples.Grabbables;
 using UnityEngine.XR.WSA.Input;
 
-public static class ExtensionMethods
+namespace HoloToolkit.Unity.InputModule.Examples.Grabbables
```
[the remark this answers is not in the document]
Comment (StephenHodgson) [reply Assets/HoloToolkit-Examples/MotionControllers-GrabMechanics/Scripts/ExtensionMethod.cs]: Unsure, I was following the established pattern.

File: Assets/HoloToolkit/UX/Scripts/Distorters/FastSimplexNoise.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (StephenHodgson) [reply Assets/HoloToolkit/UX/Scripts/Distorters/FastSimplexNoise.cs]: A fancy name?

File: Assets/HoloToolkit/UX/Scripts/Distorters/FastSimplexNoise.cs [hunk 5]
```diff
@@ -1,448 +1,589 @@
...
-             11,  4,  4,      4,  11,  4,     4,  4,  11,
-            -11, -4,  4,     -4, -11,  4,    -4, -4,  11,
-             11, -4,  4,      4, -11,  4,     4, -4,  11,
-            -11,  4, -4,     -4,  11, -4,    -4,  4, -11,
-             11,  4, -4,      4,  11, -4,     4,  4, -11,
-            -11, -4, -4,     -4, -11, -4,    -4, -4, -11,
-             11, -4, -4,      4, -11, -4,     4, -4, -11,
+        private static double[] gradients3D =
+        {
+            -11, 4, 4, -4, 11, 4, -4, 4, 11,
```
Comment (keveleigh) [inline Assets/HoloToolkit/UX/Scripts/Distorters/FastSimplexNoise.cs]: Hmm, the formatting looked nicer before this change, having each column align from top to bottom.

File: Assets/HoloToolkit/UX/Scripts/Distorters/FastSimplexNoise.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (keveleigh) [reply Assets/HoloToolkit/UX/Scripts/Distorters/FastSimplexNoise.cs]: Should this be `HoloToolkit.Unity`?

**Comment by NeerajW** [comment]
Marking blocked because we need a bigger namespace overhaul during the holidays, so let's just hold off for that work and get it right in one shot.

**Pull request closed without merging**